modules/images-cluster-sample-imagestream-import.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+// * openshift_images/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="images-cluster-sample-imagestream-import_{context}"]
+= Configuring periodic importing of Cluster Sample Operator image stream tags
+
+.Procedure
+
+. Fetch all the imagestreams in the `openshift` namespace by running the following command:
++
+[source,terminal]
+----
+oc get imagestreams -nopenshift
+----
+
+. Fetch the tags for every imagestream in the `openshift` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc get is <image-stream-name> -o jsonpath="{range .spec.tags[*]}{.name}{'\t'}{.from.name}{'\n'}{end}" -nopenshift
+----
++
+For example:
++
+[source, terminal]
+----
+$ oc get is ubi8-openjdk-17 -o jsonpath="{range .spec.tags[*]}{.name}{'\t'}{.from.name}{'\n'}{end}" -nopenshift
+----
++
+.Example output
+[source, terminal]
+----
+1.11	registry.access.redhat.com/ubi8/openjdk-17:1.11
+1.12	registry.access.redhat.com/ubi8/openjdk-17:1.12
+----
+
+. Schedule periodic import of images for each tag present in the image stream by running the following command:
```

Comment:
s/periodic import/the periodic importing

Reply:
Ack